### PR TITLE
Fixed Code Quality Issues

### PR DESCRIPTION
**Pull request by ankitdobhal**
## Description

Hi 👋 I ran the [DeepSource](https://deepsource.io/) static analyzer on the forked copy of this repo and found some [interesting code quality issues](https://deepsource.io/gh/ankitdobhal/audio/issues). This PR fixes a few of them.

## Summary of Fixes

-  Fixed Object Inheritance
-  Consider merging these comparisons with 'in'
-  Remove unnecessary use of comprehension 
-  Added .deepsource.toml to fix bug risk

## Type of change

- [✔] Antipattern
- [ ] New feature (non-breaking change which adds functionality)

## Checklist:

- [✔] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation

### CODE REVIEW

**Comment by mthrok** [comment]
Hi @ankitdobhal 

Thanks for opening the PR. The changes to code itself looks good, but we cannot add the configuration file at the moment without an approval from PyTorch. If you remove the toml file, we can merge this PR. and if you want to pursue adding the toml file, please first open an issue in https://github.com/pytorch/pytorch/issues/new/choose and get an approval.

**Comment by ankitdobhal** [comment]
> Hi @ankitdobhal
> 
> Thanks for opening the PR. The changes to code itself looks good, but we cannot add the configuration file at the moment without an approval from PyTorch. If you remove the toml file, we can merge this PR. and if you want to pursue adding the toml file, please first open an issue in https://github.com/pytorch/pytorch/issues/new/choose and get an approval.

Hi @mthrok.
Thanks for response . As per your suggestion I have opened a new Issue about   Potential Bug Risks and Anti-Patterns.
#1358

**Comment by mthrok** [comment]
> > Hi @ankitdobhal
> > Thanks for opening the PR. The changes to code itself looks good, but we cannot add the configuration file at the moment without an approval from PyTorch. If you remove the toml file, we can merge this PR. and if you want to pursue adding the toml file, please first open an issue in https://github.com/pytorch/pytorch/issues/new/choose and get an approval.
> 
> Hi @mthrok.
> Thanks for response . As per your suggestion I have opened a new Issue about Potential Bug Risks and Anti-Patterns.
> #1358

Please open it in pytorch/pytorch, not in pytorch/audio.

**Comment by ankitdobhal** [comment]
> > > Hi @ankitdobhal
> > > Thanks for opening the PR. The changes to code itself looks good, but we cannot add the configuration file at the moment without an approval from PyTorch. If you remove the toml file, we can merge this PR. and if you want to pursue adding the toml file, please first open an issue in https://github.com/pytorch/pytorch/issues/new/choose and get an approval.
> > 
> > 
> > Hi @mthrok.
> > Thanks for response . As per your suggestion I have opened a new Issue about Potential Bug Risks and Anti-Patterns.
> > #1358
> 
> Please open it in pytorch/pytorch, not in pytorch/audio.

@mthrok thanks , opened issue in official pytorch repo.
https://github.com/pytorch/pytorch/issues/53338

**Comment by ankitdobhal** [comment]
> Hi @ankitdobhal
> 
> Thanks for opening the PR. The changes to code itself looks good, but we cannot add the configuration file at the moment without an approval from PyTorch. If you remove the toml file, we can merge this PR. and if you want to pursue adding the toml file, please first open an issue in https://github.com/pytorch/pytorch/issues/new/choose and get an approval.

@mthrok I have removed the .toml file.
Now kindly merge the fixes .
Thanks

File: examples/interactive_asr/vad.py [hunk 1]
```diff
@@ -35,7 +35,7 @@ def compute_spectral_flatness(frame, epsilon=0.01):
     return -10 * torch.log10(epsilon + geometric_mean / arithmetic_mean)
 
 
-class VoiceActivityDetection(object):
+class VoiceActivityDetection():
```
Comment (mthrok) [inline examples/interactive_asr/vad.py:38]: Can you remove `()`?

File: examples/interactive_asr/vad.py [hunk 2]
```diff
@@ -148,7 +148,7 @@ def iter(self, frame):
                 return self.silence_mark
 
 
-class MicrophoneStream(object):
+class MicrophoneStream():
```
Comment (mthrok) [inline examples/interactive_asr/vad.py:151]: Can you remove `()`?

File: test/torchaudio_unittest/compliance/generate_fbank_data.py [hunk 3]
```diff
@@ -103,8 +103,8 @@ def decode(fn, sound_path, exe_path, scp_path, out_dir):
     # print args for python
     inputs['dither'] = 0.0
     logging.info(inputs)
-    sound, sample_rate = common_utils.load_wav(sound_path, normalize=False)
-    kaldi_output_dict = {k: v for k, v in torchaudio.kaldi_io.read_mat_ark(out_fn)}
+    sound, sample_rate = torchaudio.load_wav(sound_path)
```
Comment (mthrok) [inline test/torchaudio_unittest/compliance/generate_fbank_data.py:106]: `load_wav` has been removed and this no longer works. Please revert the change.

**Review by mthrok — CHANGES_REQUESTED** [review]
Please address the comments.

**Comment by ankitdobhal** [comment]
@mthrok I have created new clean PR .
#1386  now review and merge the fixes.

**Pull request closed without merging**